src/cli/rustup_mode.rs

```diff
@@ -404,6 +405,12 @@ pub fn cli() -> App<'static, 'static> {
                     SubCommand::with_name("uninstall")
                         .about("Uninstall a toolchain")
                         .alias("remove")
+                        .arg(
+                            Arg::with_name("regex")
+                                .help("Use a regular expression to select the toolchains to uninstall")
+                                .long("regex")
+                                .takes_value(false),
+                        )
```

Comment:
Side note: I noticed that rustfmt didn't format this part at all; I had to do it manually. Is that expected? Maybe the expression building up the CLI is too big?

Reply:
This chunk of code is too big for rustfmt.  We need to break it up, but that's something for another day.